paths/paths.go

```diff
@@ -23,7 +23,7 @@ var AppSharedDir syncs.AtomicValue[string]
 // or the empty string if there's no reasonable default.
 func DefaultTailscaledSocket() string {
 	if runtime.GOOS == "windows" {
-		return ""
+		return `\\.\pipe\tailscaled`
```

Comment:
Wireguard puts this in `\\.\pipe\ProtectedPrefix\Administrators\WireGuard`. I'm not sure if we should put ours under that path too. I can't find any documentation about this anywhere

e.g. `\\.\pipe\ProtectedPrefix\Administrators\tailscaled`

@dblohm7 @raggi thoughts?

Reply:
Looking...

Reply:
https://www.tiraniddo.dev/2017/11/named-pipe-secure-prefixes.html

I'm spinning up a Windows 7 VM to see what this looks like.

Reply:
@maisem I have definitely never heard of that before and it appears to be completely undocumented, but I also found that this setup works on Windows 7 exactly as described in that blog post. We might as well do that here, too.

Reply:
Done